### ISSUE TITLE
LUIS Out of call volume quota. Can't add paid key on same model.

### ISSUE THREAD

**Issue by rjgmail88** [Microsoft/BotBuilder#4639]
## Bot Info
* SDK Platform: Node.js 
* SDK Version: V3
* Active Channels: MS Teams
* Deployment Environment: Azure Bot Service, Azure App Service, local development with Emulator

## Issue Description
I have 2 LUIS apps. One is very old created 6 months back where I am using paid subscription key. I created new one just a week ago. Today it ran out of hits so I wanted to configure same subscription key for this model. 
On the LUIS [https://www.luis.ai/](https://www.luis.ai/) I can't do that anymore. I have a screenshot shot where its trying to pull **tenant ID.** I am logged in with personal MSA and don't have azure subscription tied to it.

How can I make my new model working by reusing old key ?

## Expected Behavior
How can I make my new model working by reusing old key ?

**Comment by rjgmail88** [comment]
![image](https://user-images.githubusercontent.com/32375467/40331925-b5b8f8f8-5d17-11e8-9d62-a2fec7ca8101.png)

**Comment by fanidamj** [comment]
Unfortunately it doesn't support that way. For security reason Azure syncs up with the LUIS associated with the same account.  You have to [create the keys](https://docs.microsoft.com/en-us/azure/cognitive-services/luis/azureibizasubscription#create-luis-endpoint-key) in the Azure portal before you [assign or manage keys](https://docs.microsoft.com/en-us/azure/cognitive-services/luis/azureibizasubscription) in the LUIS portal. Or you can try associating the other [Azure paid account to Active Directory of the current account](https://docs.microsoft.com/en-us/azure/active-directory/active-directory-how-subscriptions-associated-directory). Hope this helps!!

**Comment by rjgmail88** [comment]
@fanidamj , Not sure why issue is closed w/o making sure if there are no other concerns.
I took following steps so let me know if  they looks good.
1) I added my AAD account which is tied to a azure subscription as a collaborator on LUIS model. (LUIS model is westus.api.cognitive.microsoft.com created by persona MSA)
2) Then I logged in with AAD account in LUIS.ai, navigated to model , and added the key. This time it populated the fields show in the above screenshot.
3) Issues 1:- After doing so LUIS model url changed to  [https://westcentralus.api.cognitive.microsoft.com](https://westcentralus.api.cognitive.microsoft.com) **from** [https://westus.api.cognitive.microsoft.com](https://westus.api.cognitive.microsoft.com) 

is this normal ?

Issues 2:- 
When I use new Url and host a BOT locally, endpoint westcentralus.api.cognitive.microsoft.com works just fine and resolves all intents.
When I use new Url and host a BOT on azure web app, endpoint westcentralus.api.cognitive.microsoft.com sometimes fails to resolve intents. Specially from Contana channel. MS team works fine in most of the cases.

This is weird behavior. Are you aware of such things ?
